dev-tools/kubernetes/metricbeat/manifest.debug.dev.yaml

```diff
@@ -163,7 +163,8 @@ spec:
         imagePullPolicy: Never
         args: [
           "-c", "/etc/metricbeat.yml",
-          "-e"
+          "-e",
+          "-system.hostfs=/hostfs",
```

Comment:
Still, why we need this change? Let's keep this PR clean and focused on the patch itself. This ensure us that we will be able to revert it without conflicts if needed.

Reply:
this is just the Tilt manifest that I use for debugging. I have tested with both changes. Nothing seems to change. Either way, removing it was a mistake in the previous PR. I am reverting this change here.

Reply:
All these files that are changed are not relevant to the specific fix right? Can the fix stand on its own without the changes in tilt relevant files? If so then please clean this PR  and include  only the code changes that are required to fix the bug. This is not only best practice but is quite handy to handle backports, reverts etc. Imagine that we want to revert this patch for some reason, this would mean that we would revert the changes in tilt files. Is this something that we want to do? I guess no. So in that case I would suggest leaving in this PR only the code changes that are required and anything that is in dev-tools and is not blocker should be commited in follow-up. Also reviewers can only focus on one thing at a time and not in 2 different things. Smaller/standalone PRs is sth that is followed in this project since the begging ;)